t/routes-application.t

```diff
@@ -1,4 +1,3 @@
-use v6;
```

Comment:
Any reason to remove this, given we recently added it everywhere?

Reply:
Branch from the past ... added again.